dashboard/test/models/teacher_feedback_test.rb

```diff
@@ -0,0 +1,65 @@
+require 'test_helper'
+
+class TeacherFeedbackTest < ActiveSupport::TestCase
+  test 'feedback per teacher - single teacher' do
+    teacher = create :teacher
+    create :teacher_feedback, teacher: teacher
+    feedback = create :teacher_feedback, teacher: teacher
+
+    retrieved = TeacherFeedback.latest_per_teacher
+    assert_equal([feedback], retrieved)
+  end
```

Comment:
nit: this might be worth a comment explaining line 6, that we're creating two and expect to find the latest. Or it could be rewritten to be more clear without comments, perhaps by renaming the `feedback` variable to `latest_feedback` or `expected`, or using an array of both and verifying the last is found.